public_html/wp-content/mu-plugins/blocks/assets/src/organizers/block-content.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+const { Disabled }       = wp.components;
+const { Component }      = wp.element;
+const { decodeEntities } = wp.htmlEntities;
+const { __ }             = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import AvatarImage                       from '../shared/avatar';
+import { ItemTitle, ItemHTMLContent }    from '../shared/block-content';
+import GridContentLayout                 from '../shared/grid-layout/block-content';
+import './block-content.scss';
+
+class OrganizersBlockContent extends Component {
+	render() {
+		const { attributes, organizerPosts }                                     = this.props;
+		const { show_avatars, avatar_size, avatar_align, content, excerpt_more } = attributes;
+
+		return (
+			<GridContentLayout { ...this.props } >
+				{ organizerPosts.map( ( post ) =>
+					<div
+						key={ post.slug }
+						className={ classnames(
+							'wordcamp-organizer',
+							'wordcamp-organizer-' + decodeEntities( post.slug ),
+						) }
+					>
+						<ItemTitle
+							className="wordcamp-organizer-title"
+							headingLevel={ 3 }
+							title={ post.title.rendered.trim() }
+							link={ post.link }
+						/>
+
+						{ show_avatars &&
+							<div className={ classnames( 'wordcamp-organizer-avatar-container', 'align-' + decodeEntities( avatar_align ) ) }>
+								<Disabled>
+									<a href={ post.link } className="wordcamp-organizer-avatar-link">
+										<AvatarImage
+											className="wordcamp-organizer-avatar"
+											name={ decodeEntities( post.title.rendered.trim() ) || '' }
+											size={ avatar_size }
+											url={ post.avatar_urls[ '24' ] }
+										/>
+									</a>
+								</Disabled>
+							</div>
+						}
+
+						{ ( 'none' !== content ) &&
+							<ItemHTMLContent
+								className={ classnames( 'wordcamp-organizer-content-' + decodeEntities( content ) ) }
+								content={  'full' === content ? post.content.rendered.trim() : post.excerpt.rendered.trim() }
+								link={ (   'full' === content || excerpt_more ) ? post.link : null }
+								linkText={ 'full' === content ? __( 'Visit session page', 'wordcamporg' ) : __( 'Read more', 'wordcamporg' ) }
```

Comment:
Forgot to change `session` here.

Reply:
Fixed in 5f405708f3a6e6a8717f53f9b2539b9841bd6366